eth/protocols/snap/metrics.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2023 The go-ethereum Authors
+// This file is part of the go-ethereum library.
+//
+// The go-ethereum library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The go-ethereum library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
+
+package snap
+
+import (
+	metrics "github.com/ethereum/go-ethereum/metrics"
+)
+
+var (
+	IngressRegistrationError = "eth/protocols/snap/ingress/registration/error"
+	EgressRegistrationError  = "eth/protocols/snap/egress/registration/error"
+
+	_ = metrics.NewRegisteredMeter(IngressRegistrationError, nil)
+	_ = metrics.NewRegisteredMeter(EgressRegistrationError, nil)
```

Comment:
Seems weird to export the name instead of the metric here.

Reply:
fixed